### PR TITLE
[AUTO-10]: Role-based authorization

### PR DESCRIPTION
**Pull request by 0sssama**
Ticket addressed: https://linear.app/osm/issue/AUTO-10/implement-role-based-access

### CODE REVIEW

File: prisma/seed.ts [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import { PrismaClient } from "@prisma/client";
 
-import { seedSuperAdmin } from "./seed/seedSuperAdmin";
+import { seedSuperAdmin, seedSecretary } from "./seed/seedSuperAdmin";
 import { seedStudents } from "./seed/seedStudents";
 import { seedInstructors } from "./seed/seedInstructors";
 import { seedLicenseFiles } from "./seed/seedLicenseFiles";
 
 export const clerkOrgId = "org_2Vr89LWYt1FRrhR7ALcBq5HqYhC";
 export const superAdminClerkId = "user_2YBRtRz9u0fyELr5fus2ty0aOeo";
```
Comment (chukskull) [inline prisma/seed.ts:9]: is there an env for this ?

File: src/app/[locale]/dash/logout/page.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import { useEffect } from "react";
+
+export default function LogOutPage() {
+  const { signOut } = useClerk();
+
+  useEffect(() => {
+    signOut(() => window.location.replace("/"));
+  }, [signOut]);
+
+  return <span>Logging out...</span>;
```
Comment (chukskull) [inline src/app/[locale]/dash/logout/page.tsx:13]: add some styling in this

File: src/components/sections/sidebar/utils.tsx [hunk 3]
```diff
@@ -22,12 +22,28 @@ const sidebarIconProps = {
   className: "mr-3",
 };
 
+export const getFallbackSidebarLinks = (
```
Comment (chukskull) [inline src/components/sections/sidebar/utils.tsx:25]: ilike this

File: src/lib/hooks/useSetActiveOrganization.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+
```
Comment (chukskull) [inline src/lib/hooks/useSetActiveOrganization.tsx:2]: clean

File: prisma/seed.ts (hunk 1 above)
Comment (0sssama) [reply prisma/seed.ts:9]: No need for it, it's not a secret

File: src/app/[locale]/dash/logout/page.tsx (hunk 2 above)
Comment (0sssama) [reply src/app/[locale]/dash/logout/page.tsx:13]: why? it wont appear for even 1ms, its fallback while user is being logged off

**Pull request merged**